src/main/kotlin/io/spotfire/solver/Application.kt

```diff
@@ -0,0 +1,16 @@
+package io.spotfire.solver
+
+import io.spotfire.solver.plugins.*
+import io.ktor.server.application.*
+import io.ktor.server.engine.*
+import io.ktor.server.netty.*
+
+fun main() {
+  embeddedServer(Netty, port = 8086, host = "0.0.0.0", module = Application::module)
+    .start(wait = true)
```

Comment:
we might also be able to add a shutdown hook here:

```kotlin
fun main() {
  val server = embeddedServer(Netty, port = port, host = host, module = Application::module).start(wait = false)

  Runtime.getRuntime().addShutdownHook(Thread {
    server.stop(10, 60)
  })
}
```

Reply:
actually, we prob want wait = true at the end bc we want to block the main thread:

```kotlin
    val server = embeddedServer(Netty, port = 8080) {
        // Configure your Ktor application module
        module()
    }.start(wait = false)

    Runtime.getRuntime().addShutdownHook(Thread {
        server.stop(10, 10000) // Numbers indicate grace period and timeout.
    })

    server.start(wait = true)